pkg/scheduler/framework/contract/OWNERS

```diff
@@ -0,0 +1,8 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - sig-scheduling-maintainers
+reviewers:
+  - sig-scheduling
+  - sig-autoscaling-maintainers
+  - x13n
```

Comment:
I found you're not in the `sig-autoscaling-maintainers` group, so I added you separately, PTAL, whether this is as expected.